.gitignore

```diff
@@ -15,14 +15,12 @@ target/
 # Vscode Settings
 .vscode/settings.json
 
-*.iml
-
 # Python
 **/__pycache__/
 .venv
 
 # library generation
-**/output/
+output/
 **/googleapis
 library_generation/test/resources/test_monorepo_postprocessing/
```

Comment:
Do we know what's the effort to remove this line as well?

Reply:
Removed.

Reply:
This line is still here, do you plan to remove it?

Reply:
Sorry I missed it, now it's removed.